src/modules/not-found/NotFound.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div class="h-full text-center text-gray-5 py-30">
```

Comment:
Make content-min-height + Change color of bg to gray

Reply:
Will put content-min-height in page skeleton instead